src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -551,6 +550,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     if (keriaConnectUrlRecord) {
       try {
+        await Agent.agent.start(keriaConnectUrlRecord.content.url as string);
```

Comment:
This will set the agent to online. I think we need to make a change to set it to online separately at the end of the `try` clause